### DISCUSSION TITLE
What's the difference between coqui-ai/TTS and mozilla/TTS?

### DISCUSSION THREAD

**Discussion by ProgrammingLife** [coqui-ai/TTS#1292] [General]
Sorry for the stupid question but what's the difference between coqui-ai/TTS and mozilla/TTS? They are both install with the same command `pip install TTS`, they both have the same command-line syntax. Why then there are two different githubs for them?
Are they use the same models format?

**Comment by domcross** [comment]
Coqui-TTS is the successor to Mozilla-TTS: some time ago Mozilla stopped their STT and TTS project.  Coqui.ai was founded by members of the former Mozilla team.

**Reply by bernaferrari** [reply]
but now Coqui is dead, we need to wait for the next successor :(

**Reply by Simon-chai** [reply]
@bernaferrari Why you say that. I am fresh here and I think it's the most active tts project,do I missing something？

**Reply by bernaferrari** [reply]
https://coqui.ai/

**Comment by ghost** [comment]
Im a bit confused about this one as well. How do they both relate to https://commonvoice.mozilla.org/ then?  If Mozilla abandoned the project, why are they still collecting data?

**Reply by lexkoro** [reply]
CommonVoice, Mozilla-TTS and Mozilla-STT were/are completely separate projects. 
Mozilla has stopped working on TTS and STT but not on CommonVoice. 
When Mozilla stopped working on TTS/STT, Coqui continued working on those projects, as domcross said.

CommonVoice: A publicly available voice dataset
TTS: A text to speech library
STT: A speech to text library

**Comment by HarrisonSteel** [comment]
Another question on this front - the Mozilla team was previously making this an open-source system and "free to all". Is there any intention of having the better quality voices (i.e. studio ones) be free for users using API?

**Comment by sun-jin-shu** [comment]
Ladies and gentlemen, how does the recent popular Kokoro TTS compare?

**Reply by dev-karani** [reply]
sounds great off the bat g👍
